pyspark/test/bigdl/test_dl_classifier.py

```diff
@@ -0,0 +1,163 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from bigdl.nn.layer import *
+from bigdl.nn.criterion import *
+from bigdl.util.common import *
+from bigdl.models.ml_pipeline.dl_classifier import *
+from pyspark.sql.types import *
+from pyspark.context import SparkContext
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+
+class TestTensorflow():
```

Comment:
why TestTensorflow?

Reply:
naming error, fixed!

Reply:
I run the run-all.sh in the pyspark/test/dev folder. It passed without error.

Reply:
@zhichao-li Is this the way above to run tests in pyspark/test/bigdl folder?